packages/cli/src/commands/hydrogen/deploy.test.ts

```diff
@@ -22,10 +22,12 @@ import {
   createDeploy,
   parseToken,
 } from '@shopify/oxygen-cli/deploy';
+import {loadEnvironmentVariableFile} from '@shopify/oxygen-cli/utils';
```

Comment:
@aswamy curious, I saw this is using `dotenv` internally but couldn't we just rely on `import {readAndParseDotEnv} from '@shopify/cli-kit/node/dot-env';` instead?